source/brailleDisplayDrivers/albatross/_threading.py

```diff
@@ -0,0 +1,160 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2022 NV Access Limited, Burman's Computer and Education Ltd.
+
+"""Threads for Tivomatic Caiku Albatross braille display driver.
+Classes:
+- L{ReadThread}; manages reconnection retries and works as a trigger
+for read operations excluding initial connection establishment
+- L{RepeatedTimer}; timer to check periodically if data needs to be sent
+to display to keep connection. If display gets nothing within approximately
+2 seconds, it falls back to "wait for connection" state.
+"""
+
+import ctypes
+import serial
+
+from ctypes import byref
+from ctypes.wintypes import DWORD
+from logHandler import log
+from serial.win32 import (
+	ERROR_IO_PENDING,
+	EV_RXCHAR,
+	GetLastError,
+	SetCommMask
+)
+from threading import (
+	Event,
+	Thread,
+	Timer
+)
+from typing import Callable
+
+from .constants import KC_INTERVAL
+
+
+class ReadThread(Thread):
+	"""Controls most of read operations and tries to reconnect when needed."""
+
+	def __init__(
+			self,
+			readFunction: Callable[[], None],
+			disableFunction: Callable[[], None],
+			event: Event,
+			dev: serial.Serial,
+			*args,
+			**kwargs
+	):
+		"""Constructor.
+		@param readFunction: Handles read operations and reconnection.
+		@type readFunction: Callable[[], None]
+		@param disableFunction: Called on connection failure.
+		@type disableFunction: Callable[[], None]
+		@param event: Exit thread when set.
+		@type event: Event
+		@param dev: Port object.
+		@type dev: serial.Serial
+		"""

```

Comment:
This is unnecessary. this is what type hints are for.

```suggestion
		"""Constructor.
		@param readFunction: Handles read operations and reconnection.
		@param disableFunction: Called on connection failure.
		@param event: Exit thread when set.
		@param dev: Port object.
		"""
```